api/swagger.json

```diff
@@ -0,0 +1,1304 @@
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/components/schemas/Sponsor"
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Not authorized"
+          },
+          "default": {
+            "description": "unexpected error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "chapter"
+        ],
+        "summary": "Add a new food, venue or other sponsor",
+        "description": "",
+        "operationId": "addSponsorToEvent",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Sponsor"
+              }
+            }
+          },
+          "description": "Sponsor details",
+          "required": true
+        },
+        "parameters": [
+          {
+            "name": "chapterId",
+            "in": "path",
+            "description": "ID of chapter to return",
+            "required": true,
+            "schema": {
+              "type": "integer",
+              "format": "int64"
+            }
+          },
+          {
+            "name": "eventId",
+            "in": "path",
+            "description": "ID of event to return",
+            "required": true,
+            "schema": {
+              "type": "integer",
+              "format": "int64"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Sponsor"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid sponsor object"
+          },
+          "401": {
+            "description": "Not authorized"
+          },
+          "404": {
+            "description": "Event/Chapter not found"
+          },
+          "default": {
+            "description": "unexpected error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/chapters/{chapterId}/events/{eventId}/sponsors/{sponsorId}": {
+      "get": {
+        "tags": [
+          "chapter"
+        ],
+        "summary": "Retreive list of sponsors by event",
```

Comment:
Looks to be a copy-paste mistake as this is an entity endpoint rather than a collection endpoint.